PG-1787 Release Notes 15

Andriciuc opened the pull request.

Andriciuc committed:
PG-1787 Release Notes 15

ohrytsenko commented on docs/templates/pdf_cover_page.tpl:
```diff
@@ -7,5 +7,5 @@
 <h1>{{ config.site_description }}</h1>
 {% endif %} 
-<h2>15.13 Update (July 14, 2025)</h2>
+<h2>15.14 (August 30, 2025)</h2>
```
hi @Andriciuc 

could you please share the updated release date of 15.14 support?

Andriciuc replied:
Hello @ohrytsenko! The update is scheduled to be released by end of next week. As soon as I have a confirmation on the exact date, I will update the above file and I will tag you as well in this discussion to update you pronto, thank you!

ohrytsenko replied:
thanks a lot

Andriciuc replied:
Hello @ohrytsenko! We are planning to release 15.14 sometime around Monday, next week. :)

ohrytsenko replied:
thanks

Andriciuc committed:
release updates

- add sboms.md
- add minor/major update notes
- update toc for sboms
- update dates
- update tab ver

Andriciuc committed:
update sboms with correct links
